cancer_ai/utils/config.py

```diff
@@ -131,77 +131,72 @@ def add_args(cls, parser):
 
 def add_miner_args(cls, parser):
     """Add miner specific arguments to the parser."""
-
     parser.add_argument(
-        "--neuron.name",
+        "--competition-id",
         type=str,
-        help="Trials for this neuron go in neuron.root / (wallet_cold - wallet_hot) / neuron.name. ",
-        default="miner",
+        help="Competition ID",
     )
 
     parser.add_argument(
-        "--blacklist.force_validator_permit",
-        action="store_true",
-        help="If set, we will force incoming requests to have a permit.",
-        default=False,
+        "--model-dir",
+        type=str,
+        help="Path for for loading the starting model related to a training run.",
+        default="./models",
     )
 
     parser.add_argument(
-        "--blacklist.allow_non_registered",
-        action="store_true",
-        help="If set, miners will accept queries from non registered entities. (Dangerous!)",
-        default=False,
+        "--hf-repo-id",
+        type=str,
+        help="Hugging Face model repository ID",
+        default="",
     )
 
     parser.add_argument(
-        "--wandb.project_name",
+        "--hf-model-name",
         type=str,
-        default="template-miners",
-        help="Wandb project to log to.",
+        help="Name of the model to push to hugging face.",
     )
 
     parser.add_argument(
-        "--wandb.entity",
-        type=str,
-        default="opentensor-dev",
-        help="Wandb entity to log to.",
+        "--action",
+        choices=["submit", "evaluate", "upload"],
     )
 
     parser.add_argument(
-        "--competition.entity",
+        "--model-path",
         type=str,
-        default="opentensor-dev",
-        help="Wandb entity to log to.",
+        help="Path to ONNX model, used for evaluation",
     )
 
     parser.add_argument(
-        "--models.load_model_dir",
+        "--dataset-dir",
         type=str,
-        help="Path for for loading the starting model related to a training run.",
-        default="./models",
+        help="Path for storing datasets.",
+        default="./datasets",
     )
 
     parser.add_argument(
-        "--models.namespace",
+        "--hf-token",
         type=str,
-        help="Namespace where the model can be found.",
-        default="mock-namespace",
+        help="Hugging Face API token",
+        default="",
     )
 
     parser.add_argument(
-        "--models.model_name",
-        type=str,
-        help="Name of the model to push to hugging face.",
-        default="mock-name",
+        "--clean-after-run",
+        action="store_true",
+        help="Whether to clean up (dataset, temporary files) after running",
+        default=False,
     )
 
     parser.add_argument(
-        "--models.epoch_checkpoint",
-        type=int,
-        help="The epoch number to submit as your checkpoint to evaluate e.g. 10",
-        default=10,
+        "--code-directory",
```

Comment:
we should make sure that all the other arguments in config.py are-named-like-this not_like_this

Reply:
yep